src/modules/commander/PreflightCheck.cpp

```diff
@@ -477,7 +477,7 @@ static bool powerCheck(orb_advert_t *mavlink_log_pub, vehicle_status_s &status,
 						mavlink_log_critical(mavlink_log_pub, "Preflight Fail: Avionics Power low: %6.2f Volt", (double)avionics_power_rail_voltage);
 					}
 
-				} else if (avionics_power_rail_voltage < 4.9f) {
+				} else if (avionics_power_rail_voltage < 4.75f) {
```

Comment:
@davids5 can you comment on this?

Reply:
This would indicate a large drop on the summing diode or a Power Module that is not outputting at 5.25V.  

This change should not be upstreamed (it has been rejected before).  We should discuss viable options.

Reply:
Hmm, let me tie in @AlexKlimaj on this